src/actions/index.js

```diff
@@ -46,25 +45,28 @@ const ALERT_LEVEL_ERROR = 'error'
 export const downloadFileMissing = () => ({ type: DOWNLOAD_FILE_E_MISSING, alert: { message: 'error.download_file.missing', level: ALERT_LEVEL_ERROR } })
 export const downloadFileOffline = () => ({ type: DOWNLOAD_FILE_E_OFFLINE, alert: { message: 'error.download_file.offline', level: ALERT_LEVEL_ERROR } })
 
-export const openFolder = (folderId, context = FILES_CONTEXT) => {
+export const locationChange = (virtualRoot, folderId) => {
```

Comment:
Regarding precisely `LOCATION_CHANGE`, it's clearly ill-named, but your suggestion is worse because the location has already changed. The action's type must reflect the consequence of the location change (a view refresh in a sense), but I can't find a better name. Any suggestions people?